library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/MediaRelaySetsRelBrandDoctrineRepository.php

```diff
@@ -30,4 +31,40 @@ public function __construct(
             $entityPersisterInterface
         );
     }
+
+    /** @return int[] */
+    public function getMediaRelaySetIdsByBrandAdmin(AdministratorInterface $admin): array
+    {
+        if (!$admin->isBrandAdmin()) {
+            throw new \DomainException('User must be brand admin');
+        }
+
+        $brandId = $admin->getBrand()?->getId();
+        if (is_null($brandId)) {
+            return [];
+        }
```

Comment:
Same here. Validations are better outside in the logic

Reply:
This is called from API, so I don't have a chance to work on the caller side.